docs/install.rst

```diff
@@ -44,6 +44,22 @@ parentheses. When the source interface is displayed in French (for
 instance), people submitting documents will expect a journalist fluent
 in French is available to read them and followup.
 
+Custom logo for the source and journalist interface
+---------------------------------------------------
+
+An image to replace the SecureDrop logo on the *Source Interface* and
+*Journalist Interface* can be provided during the installation. It should
+be prepared in advance and available from the *Admin Workstation*, either
+via an URL or from the persistent tails directory.
```

Comment:
can we remove "either via an URL or from the persistent tails directory." - I feel like people _might_ get confused and think they can provide just a URL instead of downloading the image

otherwise this all looks excellent! (and this is an extremely common source of confusion, so these docs will be really helpful :tada: 💃 )